### ISSUE TITLE
No permission to export

### ISSUE THREAD

**Issue by Beginner131** [processing/processing#5525]
Im new in programming with Processing, but I wrote a small programm right now.
But if I try to export it (using Win 10), I get an error:
...\launch4j\bin\ld.exe: cannot open output file ...\sketch_180604a.exe: Permission denied

Do you know what I can do to run the export successfully?

**Comment by gohai** [comment]
What's the full path where the sketch you're trying to export is located? (Is it, potentially, in a part of the file system that regular users can't write to - such as the `Programs` directory?)

**Comment by Beginner131** [comment]
Processing and its projects are saved in my documents ...
That is the full error message:

Compiling resources
Linking
C:\Users\myName\Documents\processing-3.3.7-windows64\processing-3.3.7\modes\java\application\launch4j\bin\ld.exe: cannot open output file C:\Users\myName\Documents\Processing Games\ReflexTraining\sketch_180604a\sketch_180604a\application.windows32\sketch_180604a.exe: Permission denied

... and more than 100 errors with Java / Java libraries

PS: I have the latest Java version and the latest JDK version.

**Comment by gohai** [comment]
@Beginner131 Sorry that you are having this issue. I haven't seen a similar `Permission denied` before.

A couple of things you could try to check:
* does the file `C:\Users\myName\Documents\processing-3.3.7-windows64\processing-3.3.7\modes\java\application\launch4j\bin\ld.exe` exist?
* when you go to `C:\Users\myName\Documents\Processing Games\ReflexTraining\sketch_180604a\sketch_180604a`, can you, at this location, create e.g. a new folder in the file system - without having to use administrator privileges (via this popup that comes up sometimes)?

**Comment by Beginner131** [comment]
@gohai 
I'm very thankful that you are trying to help me.
But I found the issue by myself; I tried to export the files in a directory where I had exported the file already.
I thought that the IDE overwrites the old directory, but it does not.
My other problem was that i can't open the exported files, but my virus protection moved it into the quarantine; it says that the files are dangerous.

Thank you so much!
